lib/amo-client.js

```diff
@@ -151,6 +148,75 @@ Client.prototype.waitForSignedAddon = function(statusUrl, opt) {
 };
 
 
+Client.prototype.downloadSignedFiles = function(signedFiles, options) {
+  options = _.assign({
+    createWriteStream: fs.createWriteStream,
+    request: this._request,
+    stdout: process.stdout,
+  }, options);
+  var self = this;
+  var allDownloads = [];
+  var dataExpected = 0.1;
```

Comment:
Why do these start with `0.1`? I assumed it would be to avoid dividing by zero when rendering the downloaded percentage but won't this mean it says `Downloading: 100%` and then when it gets the response it changes to `Downloading: 0%`? Maybe I'm missing something but that seems like it is worth handling.